options.go

```diff
@@ -100,3 +100,27 @@ func (o *FetchOptions) Validate() error {
 
 	return nil
 }
+
+// PushOptions describe how a push should be performed.
+type PushOptions struct {
+	// RemoteName is the name of the remote to be pushed to.
+	RemoteName string
+	// RefSpecs specify what destination ref to update with what source
+	// object. A refspec with empty src can be used to delete a reference.
+	RefSpecs []config.RefSpec
```

Comment:
yes: refs/heads/*:refs/heads/*. I've just added it. Note that git has different behaviours for default push refspec depending on config.